components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementAdminService.java

```diff
@@ -51,8 +51,13 @@
      */
     public void createApplication(ServiceProvider serviceProvider)
             throws IdentityApplicationManagementException {
-        applicationMgtService = ApplicationManagementService.getInstance();
-        applicationMgtService.createApplication(serviceProvider, getTenantDomain(), getUsername());
+        try {
+            applicationMgtService = ApplicationManagementService.getInstance();
+            applicationMgtService.createApplication(serviceProvider, getTenantDomain(), getUsername());
+        }catch(IdentityApplicationManagementException idpException){
```

Comment:
Please fix the formatting issues. There are some in the below as well. Please fix all occurrences.